opsdroid/connector/matrix/connector.py

```diff
@@ -224,18 +237,24 @@ async def get_nick(self, roomid, mxid):
         """
         if self.room_specific_nicks:
             try:
-                return await self.connection.get_room_displayname(roomid, mxid)
+                res = await self.connection.get_displayname(mxid)
+                if isinstance(res, nio.ProfileGetDisplayNameError):
+                    logging.exception("Failed to lookup nick for %s.", mxid)
+                    return mxid
+                if res.displayname is None:
+                    return mxid
+                return res.displayname
             except Exception:  # pylint: disable=W0703
                 # Fallback to the non-room specific one
                 logging.exception("Failed to lookup room specific nick for %s.", mxid)
```

Comment:
The exception is being dealt with within the try clause itself, plus I don't think the try clause would actually throw any exceptions at all.

Reply:
I think you're right. I'll remove the exception